notification-portlet-webapp/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to Jasig under one or more contributor license
+    agreements. See the NOTICE file distributed with this work
+    for additional information regarding copyright ownership.
+    Jasig licenses this file to you under the Apache License,
+    Version 2.0 (the "License"); you may not use this file
+    except in compliance with the License. You may obtain a
+    copy of the License at:
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on
+    an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+<persistence version="2.0"
+  xmlns="http://java.sun.com/xml/ns/persistence" 
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://java.sun.com/xml/ns/persistence 
+  http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd">
+
+    <persistence-unit name="NoticePU" transaction-type="RESOURCE_LOCAL">
+        <provider>org.hibernate.ejb.HibernatePersistence</provider>
+        <class>org.jasig.portlet.notice.service.jpa.JpaAction</class>
+        <class>org.jasig.portlet.notice.service.jpa.JpaAddressee</class>
+        <class>org.jasig.portlet.notice.service.jpa.JpaAttribute</class>
+        <class>org.jasig.portlet.notice.service.jpa.JpaEntry</class>
+        <class>org.jasig.portlet.notice.service.jpa.JpaEvent</class>
+        <properties>
+            <!-- property name="hibernate.ejb.cfgfile" value="/notification.cfg.xml" /-->
+            <property name="hibernate.session_factory_name_is_jndi" value="false" />
+            <property name="hibernate.format_sql" value="true" />
+            <property name="hibernate.generate_statistics" value="true" />
```

Comment:
Adding this to make it easy to turn on better debugging would be a nice addition:
            
```
<!-- For additional detail/debugging 
            <property name="hibernate.show_sql" value="true" />
            <property name="hibernate.use_sql_comments" value="true"/>
 -->
```